MAUI/Scheduler/getting-started.md

```diff
@@ -602,4 +602,6 @@ this.Content = scheduler;
 
 N> It is not applicable if the `View` is `Timeline Month` and it is applied only when the `ShowWeekNumber` property is `enabled.`
 
-![customize-the-week-number-text-style-in-maui-scheduler](images/getting-started/customize-the-week-number-text-style-in-maui-scheduler.png)
\ No newline at end of file
+![customize-the-week-number-text-style-in-maui-scheduler](images/getting-started/customize-the-week-number-text-style-in-maui-scheduler.png)
+
+N> You can refer to our [.NET MAUI Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Scheduler Example](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main) that shows you how to render the Scheduler in .NET MAUI.
```

Comment:
Use the below demo link
https://github.com/syncfusion/maui-demos/tree/master/MAUI/Scheduler

Reply:
Updated